manifests/5.2/cluster-logging.clusterserviceversion.yaml

```diff
@@ -325,6 +325,10 @@ spec:
                     value: "cluster-logging-operator"
                   - name: FLUENTD_IMAGE
                     value: "quay.io/openshift/origin-logging-fluentd:latest"
+                  - name: CURATOR_IMAGE
```

Comment:
Sorry I broke you here.  @vimalk78 asked me to move the file name so will need a rebase once https://github.com/openshift/cluster-logging-operator/pull/1013 merges